python/lsst/jointcal/utils.py

```diff
@@ -17,13 +17,16 @@
 
 import lsst.log
 import lsst.afw.table
+from lsst.afw.image import fluxFromABMag, abMagFromFlux
 from lsst.afw.geom import arcseconds
 
+MatchDict = collections.namedtuple('MatchDict', ['relative', 'absolute'])
+
 
 class JointcalStatistics(object):
     """Compute statistics on jointcal-processed data, and optionally generate plots."""
```

Comment:
can this be done quickly by attaching the attributes to private getter methods? Then you can intercept the request for the statistic and calculate it on demand (storing the result in an private attribute for next time it's requested).